packages/views.api.unstable/src/manifest.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2024 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type {
+  EventDefinition,
+  ParameterConfig,
+  ParameterDefinition,
+} from "./config.js";
+
+export interface ViewManifest {
+  views: Record<string, ViewManifestConfig>;
```

Comment:
What does the key represent here and is it meaningful? It looks like in the plugin we take the chunk that contains the config name so is it intended to just be a random unique identifier for the view?

```
          viewConfigManifest.views[chunk.name] = viewConfig;
```

Should we ask the user to pick a unique id key themselves in the config? Although we don't actually have that as a real concept of a view id outside of here

Reply:
it's not meaningful, it's mostly for bookkeeping inside the vite plugin so i can keep track of individual entrypoints. if someone were to handroll this, they could type whatever they want for that key